app/views/addresses/edit.html.haml

```diff
@@ -8,11 +8,11 @@
   = render partial: 'form',
            locals: { address: @address, method: :put,
                      url: company_address_path(@company.id, @address.id) }
-  .center
-    = link_to "#{t('companies.view_company')}", @company
+  %hr
+  .text-center
```

Comment:
We could also change .text-center if needed. 
But since they were .center before and that is not used anymore I just updated it to text-center which is the new variant of just that. 

I do agree though, just didnt think about doing that =)

Reply:
I think this is a separate story to do, it can/should be done with page-headers as well, because they are also not very DRY ^_-

Reply:
Agreed:  we should probably just have a separate PT story for DRYing up the css, and 1 or more stories for some partials that could be created for DRYing up the views.  
And they can and should be done _after_ this one is merged.